src/resources/views/base/widgets/alert.blade.php

```diff
@@ -1,9 +1,13 @@
 @includeWhen(!empty($widget['wrapper']), 'backpack::widgets.inc.wrapper_start')
 
```

Comment:
Hmm when I see alerts, they don't really have any color on them:

![CleanShot 2023-02-26 at 10 57 20@2x](https://user-images.githubusercontent.com/1032474/221401285-6724baeb-7e7e-48a7-8a9c-05cb773a2d9a.png)

Reply:
In addition to that, `pace` was not working. It should be ok now.